graph/patterns/query-option-aliasing.md

```diff
@@ -0,0 +1,119 @@
+# Query option aliasing
```

Comment:
>Query option aliasing [](http://example.com/codeflow?start=2&length=21)

I'm not sure that the name "Query option aliasing" lands for me, although I understand your intent.

Maybe something like "Exposing functions for common queries"?

Reply:
I didn't remove *all* references to the work alias, but I made some changes, let me know if you think I need to go deeper